kvbc/include/reconfiguration_kvbc_handler.hpp

```diff
@@ -125,4 +125,18 @@ class InternalKvReconfigurationHandler : public concord::reconfiguration::IRecon
               uint64_t bft_seq_num,
               concord::messages::ReconfigurationResponse&) override;
 };
+
+class InternalPostKvReconfigurationHandler : public concord::reconfiguration::IReconfigurationHandler,
+                                             public ReconfigurationBlockTools {
+ public:
+  InternalPostKvReconfigurationHandler(kvbc::IBlockAdder& block_adder, kvbc::IReader& ro_storage)
+      : ReconfigurationBlockTools{block_adder, ro_storage} {}
+  bool verifySignature(uint32_t sender_id, const std::string& data, const std::string& signature) const override {
```

Comment:
no need to have a trivial implementation here

Reply:
apparently verifySignature is pure virutal
```// The verification method is pure virtual as all subclasses has to define how they verify the reconfiguration
  // requests.
  virtual bool verifySignature(uint32_t sender_id, const std::string& data, const std::string& signature) const = 0;```